etc/samples/sds_objects_generic.sample.yaml

```diff
@@ -0,0 +1,37 @@
+# This schema is not used stand-alone, it is a sub-section of "tendrl_definitions_<sds_nvr>.yaml" | "tendrl_definitions_generic.sample.yaml
+
+---
+valid_objects:
+- volume
+
+object_details:
+  volume:
+    enabled: True
+    value: "/clusters/gluster/$cluster.id/volumes/$volume.id
```

Comment:
That makes sense. But in reference to @shtripat comments about uuid above, I would prefer mapping the tendrl world cluster uuid's to their sds_type and other details at 

eg: assume 1 gluster cluster and 1 ceph cluster already created/imported
- /clusters/gluster/12321321/
- /clusters/ceph/32431432/

You can then assume to find mappings of above tendrl data at 
- /clusters/tendrl_context: {"12321321" : {sds_type: "gluster-<nvr>", rest of the metadata},  "32431432": {"sds_type": "ceph-<nvr>", rest of the metadata}

works for you?